### DISCUSSION TITLE
Output HEVC stream to Youtube using Enhanced RTMP

### DISCUSSION THREAD

**Discussion by jahaddow** [bluenviron/mediamtx#3708] [Questions]
### Question

I am ingesting a HEVC stream using SRT on mediamtx. How can I output this to YouTube via RTMP? I have played around with Paths and FFmpeg but cant seem to get it working.

**Comment by jahaddow** [comment]
Any reason why you can publish H265 via RTMP on mediamtx but can't read it? Can only read H264 via RTMP

**Reply by aler9** [reply]
this is because currently there are no players that support reading H265 through RTMP.

**Comment by aler9** [comment]
Instructions on how to forward streams to another server are in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#forward-streams-to-other-servers

[a comment by an automated account is not shown]
